crates/net/network/Cargo.toml

```diff
@@ -19,7 +19,7 @@ normal = [
 [dependencies]
 # reth
 reth-interfaces.workspace = true
-reth-primitives.workspace = true
+reth-primitives = { workspace = true, features = ["test-utils"] }
```

Comment:
This should be enabled just for dev-dependencies

Reply:
@DaniPopes Done